archetypes/helidon/src/main/archetype/mp/database/files/src/main/resources/hibernate.properties.mustache

```diff
@@ -1,3 +1,8 @@
 # Byte code for JPA must be generated at compile time.
 # This is a limitation of native image
-hibernate.bytecode.provider=none
+hibernate.bytecode.use_reflection_optimizer=false
+hibernate.connection.provider_class=org.hibernate.hikaricp.internal.HikariCPConnectionProvider
+hibernate.hikari.dataSourceClassName={{jdbcDataSource}}
+hibernate.hikari.dataSource.url={{databaseUrl}}
+hibernate.hikari.dataSource.user={{dbUser}}
+hibernate.hikari.dataSource.password={{password}}
```

Comment:
Hmm; could you say why you needed to do this? Shouldn't be needed, but maybe you had a good reason to bypass our data source integration.